frontend/src/pages/SessionForm/SessionForm.js

```diff
@@ -236,7 +229,7 @@ function SessionForm({ onSendSessionToBackend }) {
           </div>
 
           <div className="sessionFormButtons">
-            <Button name="Save" onClick={handleSubmit(onSaveSession)} />
+            <Button name="Save" onClick={() => onSaveSession()} />
```

Comment:
Clicking the SAVE button now reloads the page. After the reload, an empty session form is shown.